pkg/commands/git_commands/rebase.go

```diff
@@ -156,6 +156,15 @@ func (self *RebaseCommands) EditRebase(branchRef string) error {
 	}).Run()
 }
 
+func (self *RebaseCommands) EditRebaseFromBaseCommit(targetBranchName string, baseCommit string) error {
+	self.os.LogCommand(fmt.Sprintf("Beginning interactive rebase from '%s' onto '%s", baseCommit, targetBranchName), false)
```

Comment:
This needs i18n

Reply:
Does it? All the existing `LogCommand` calls I could find use literal strings.

Reply:
Wow right you are. That's a huge oversight on my part. My rule has been that anything which the user can see within lazygit should be i18n'd. That means that logs that go to the log file (e.g. `self.c.Log.Warnf(...)`) don't need to i18n. But `LogCommand` and `LogAction` do.

Could you i18n this one and then I'll raise an issue for the rest?

Reply:
Issue: https://github.com/jesseduffield/lazygit/issues/2848

Reply:
I'll wait with this until after #2852 is merged, so that I know where to put the new text.

Reply:
OK, I changed my mind. 😄 Since this PR was otherwise ready, I decided to merge it now. @KarlHeitmann, can you please take care of this command too in your #2852 PR? Thanks!

Reply:
Ok! I will fix it tonight.